src/wp-includes/rest-api.php

```diff
@@ -820,6 +820,39 @@ function rest_filter_response_fields( $response, $server, $request ) {
 
 	$fields = wp_parse_list( $request['_fields'] );
 
+	$embed = isset( $request['_embed'] ) ? rest_parse_embed_param( $request['_embed'] ) : false;
+
+	if ( $embed && ! isset( $fields['_links'] ) ) {
```

Comment:
Should this be an `in_array` check instead? `_fields` is a list, right?

```suggestion
	if ( $embed && ! in_array( '_links', $fields, true ) ) {
```

Reply:
I thought that `isset` is faster than `in_array`

Reply:
It is, but they don't do the same thing. `$fields` is something like `[ 'id', '_links' ]`. So to check that it contains `_links` we need to use `in_array`.

If `$fields` was something like `[ 'id' => true, '_links' => true ]`, then we could use `isset`.

Reply:
@TimothyBJacobs Done